src/utils/SearchBoxResize.ts

```diff
@@ -0,0 +1,57 @@
+import { $$ } from './Dom';
+import _ = require('underscore');
+
+export class SearchBoxResize {
+  public static resize(element: HTMLElement, size: number) {
+    const searchBox = this.findsearchBox(element);
+    const InputChange: HTMLElement[] = this.findOnlyHeigthChange(element);
+    const magicBoxChange: HTMLElement[] = this.findMagicBoxElement(element);
+    const magicBoxInput: HTMLElement = this.findMagicBoxInput(element);
+
+    const searchBoxStyle = `height: ${size}px; line-height: ${size}px; display: inline-block; align-items: center;`;
```

Comment:
I understand you can't pass some of those CSS value to the SASS (those using the `size` variable)
But you could put the rest inside the SASS using a class in order to have the less inline CSS possible

Reply:
yeah, that's a great idea. I'll do that